tests/instance-interface-syntax-tests.dx

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Any particular reason to write these tests as file-check tests rather than quine tests? I'm mostly just curious. Either is fine, but I find the quine tests make it clearer what the expected output can be, even if they're a bit more brittle.

Reply:
No deep reason. I wrote a file-check test mostly because I am familiar with that tool.